schedule/schedule_commands.go

```diff
@@ -479,13 +479,13 @@ func DescribeSchedule(c *cli.Context) error {
 	if fields := resp.Memo.GetFields(); len(fields) > 0 {
 		item.Memo = make(map[string]string, len(fields))
 		for k, payload := range fields {
-			item.Memo[k] = dataconverter.CustomDataConverter().ToString(payload)
+			item.Memo[k] = converter.GetDefaultDataConverter().ToString(payload)
```

Comment:
Correctness: using the same `resp` as [above](https://github.com/temporalio/cli/pull/384/files?show-deleted-files=true&show-viewed-files=true&file-filters%5B%5D=#r1406065941).